provider/app.py

```diff
@@ -157,6 +157,14 @@ def main():
     streamHandler.setFormatter(formatter)
     logger.addHandler(streamHandler)
 
+    # also log to file if /logs is present
+    if os.path.isdir('/logs'):
+        fh = logging.FileHandler('/logs/kafkatriggers.log')
```

Comment:
I believe the file name needs to be predictable. @jasonpet do you want to comment?